half-light.js

```diff
@@ -91,6 +91,13 @@
     });
   })
   
+  let script = document.currentScript;
+  document.addEventListener("DOMContentLoaded", () => {
+    if(script.hasAttribute('disable-live-half-light')) {
+      observer.disconnect()
```

Comment:
The key here is to clear `openStylableElements` and separately disable adding to the map. The concern with the live updates is not the perf of the observer, it's the memory "leak" of maintaining elements in the `openStylableElements` Set.